src/analytics/sql/changesets.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE changesets (
+    id bigint NOT NULL,
+    road_km_added double precision,
```

Comment:
Would it be OK to mark all columns that we can reliably expect to be there as NOT NULL? It makes me feel more comfortable writing up the models that will be materialized from these tables if I can give Option a consistent semantics.
Seems to me we can initialize/fallback to 0 as needbe for many of these.

Reply:
Hmm. Yes and no. Since there are 2 processes that update individual rows (minutely diffs + changeset diffs), different parts of the data may be uninitialized and it's potentially useful to know which is which.

Defaulting to 0 could work, but with the not knowing how it was initialized caveat.

Reply:
Makes sense. I'll work to match you on this since my side of things is quite a bit simpler, I think